src/overlays/actors/ovl_Obj_Warpstone/z_obj_warpstone.h

```diff
@@ -5,11 +5,30 @@
 
 struct ObjWarpstone;
 
+typedef s32 (*ObjWarpstoneActionFunc)(struct ObjWarpstone* this, GlobalContext* globalCtx);
+
+typedef enum {
+    /* 0 */ SEK_MODEL_CLOSED,
+    /* 1 */ SEK_MODEL_OPENED
+} ObjectSekModels;
+
+typedef enum {
+    /* 25 */ OBJ_WARPSTONE_TIMER_OPENED = 25,
+    /* 66 */ OBJ_WARPSTONE_TIMER_DONE = 66
+} ObjWarpstoneTimerCutoffs;
+
 typedef struct ObjWarpstone {
-    /* 0x000 */ Actor actor;
-    /* 0x144 */ char unk_144[0x6C];
+    /* 0x000 */ DynaPolyActor dyna;
```

Comment:
I'm not sure if dyna is the best option here. I know the next thing starts at `0x15C`, but that could definitely be a coincidence since plenty of actors (especially, seemingly, the small ones?) have a bunch of random holes/padding in them. Not a huge deal either way, but since the dyna features aren't used in this actor, I'm not sure the best course here.

Reply:
Yeah me either. The struct looks a lot cleaner as a dyna, but the flipside is it does add a bunch of struct element selections to the code.

Reply:
This is a tough one. It can literally be anything so its hard to say.
The only thing there really is to go off of here is the actor name prefixes. They are by no means consistent with them, but for what is worth: there are 27 actors in oot prefixed with `obj_`, and 10 of them are dynapoly. If this were a bg_ actor id be more inclined to say that it was definitely dyna

Only other suggestion i have is to have a look at the object file for this actor and see if there are any unused collision headers. If there is one, then it 100% used to be dyna at some point in development

Reply:
There could be some other unused asset in the object as well that could explain the type of the data. Otherwise my preference would be that we just give it padding.
```suggestion
    /* 0x000 */ Actor actor;
    /* 0x144 */ UNK_TYPE1 pad_144[0x10];
```

Reply:
How would I find an unused collision header? When I'm looking through the object file in z64Utils, I only see dlist, unk, tlut, tex, and vtx.